package.json

```diff
@@ -51,4 +52,4 @@
     "url": "https://github.com/wikimedia/html-metadata/issues"
   },
   "homepage": "https://github.com/wikimedia/html-metadata"
-}
+}
```

Comment:
The red circle with the line through it means this file is missing a new line character. 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline